modules/oe_content_call_proposals/src/Plugin/Validation/Constraint/CallProposalsJournalTitleLengthValidator.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_call_proposals\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Drupal\Component\Utility\Unicode;
+
+/**
+ * Validates the Call For Proposals journal title length constraint.
+ */
+class CallProposalsJournalTitleLengthValidator extends ConstraintValidator {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validate($node, Constraint $constraint) {
+    /* @var \Drupal\node\NodeInterface $node */
+    if (!isset($node) || $node->getType() !== 'oe_call_proposals') {
+      return;
+    }
+
+    $length = 128;
+
+    $title = $node->oe_call_proposals_journal->title;
+    if (!empty($title) && Unicode::strlen($title) > $length) {
```

Comment:
from my tests I was able to add a string of 129, then I tried with 134 long and it was ok, max is 128, not sure what is going on

Reply:
I tested with a string with 129 characters and the validation error was triggered. Can you please check again?

Reply:
its ok